### ISSUE TITLE
how to use native rooting in Marshmallow 6

### ISSUE THREAD

**Issue by Xardas999** [ukanth/afwall#524]
The new smartphones like Huawei 8 Mate with the Android Marshmallow have a natively implemented feature to delegate the root access to selected applications, Setting/More/Security/Device administrators.

Can we use the feature without the need to root the device?
The Afwall appeared in the list there, and I switched "allow", BUT the list of the root permission is empty and the app says the device isn't rooted yet.

**Comment by CHEF-KOCH** [comment]
Yes, the integrated iptables and VPN rules are fine to go without anything. You can control it directly via GUI you not get any root popup as long you not want to manipulate via script or terminal the iptables directly.

The popup only comes if you really 'work' with AFWall+, which includes iptables related changes. Marshmellow by default offers already an basic firewall GUI which is normally enough. AFWall+ just kicks in if you e.g. want to block a specific range of ip's or if you want to work with certain conditions.
